### PR TITLE
Add MPL license to files

### PR DESCRIPTION
**Pull request by dguenther**
Adds the MPL license to files, same as how is done in https://github.com/iron-fish/ironfish/blob/master/LICENSE.

Fixes IFL-1130

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -29,6 +29,16 @@
         "argsIgnorePattern": "^_"
       }
     ],
+    "header/header": [
+      2,
+      "block",
+      [
+        " This Source Code Form is subject to the terms of the Mozilla Public",
+        " * License, v. 2.0. If a copy of the MPL was not distributed with this",
+        " * file, You can obtain one at https://mozilla.org/MPL/2.0/. "
```
Comment (dgca) [inline .eslintrc:38]: nitpick: Super nit, but I think we could shorten this to "SPDX-License-Identifier: MPL-2.0".

**Review by dgca — APPROVED** [review]

File: .eslintrc (hunk 1 above)
Comment (dguenther) [reply .eslintrc:38]: I think this is a good idea, but for consistency's sake I'd want to do it in the main ironfish repository to start with. Searching around a bit, it might be a good idea to add a copyright line as well
